cypress/e2e/finance/funds/transfer-allocation-between-funds.cy.js

```diff
@@ -0,0 +1,248 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import {
+  FiscalYears,
+  FinanceHelper,
+  Ledgers,
+  Funds,
+  Budgets,
+  Transfers,
+} from '../../../support/fragments/finance';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+
+describe('Finance', () => {
+  const testData = {
+    user: {},
+  };
+
+  let fiscalYear;
+  let ledger;
+  let funds;
+  let budgets;
+
+  const createTestFunds = ({ secondBudget = 100 } = {}) => {
+    fiscalYear = FiscalYears.getDefaultFiscalYear();
+    ledger = { ...Ledgers.getDefaultLedger(), fiscalYearOneId: fiscalYear.id };
+    funds = {
+      first: { ...Funds.getDefaultFund(), ledgerId: ledger.id },
+      second: { ...Funds.getDefaultFund(), ledgerId: ledger.id },
+    };
+    budgets = {
+      first: {
+        ...Budgets.getDefaultBudget(),
+        allocated: 100,
+        fiscalYearId: fiscalYear.id,
+        fundId: funds.first.id,
+      },
+      second: {
+        ...Budgets.getDefaultBudget(),
+        allocated: secondBudget,
+        fiscalYearId: fiscalYear.id,
+        fundId: funds.second.id,
+      },
+    };
+
+    cy.getAdminToken().then(() => {
+      FiscalYears.createViaApi(fiscalYear);
+      Ledgers.createViaApi(ledger);
+      Object.values(funds).forEach((fund) => {
+        Funds.createViaApi(fund);
+      });
+      Object.values(budgets).forEach((budget) => {
+        Budgets.createViaApi(budget);
+      });
+    });
+  };
+
+  before('Create test user', () => {
+    cy.createTempUser([
+      Permissions.uiFinanceCreateTransfers.gui,
+      Permissions.uiFinanceViewFundAndBudget.gui,
+    ]).then((userProperties) => {
+      testData.user = userProperties;
+    });
+  });
+
+  after('Delete test user', () => {
+    cy.getAdminToken().then(() => {
+      Users.deleteViaApi(testData.user.userId);
```

Comment:
don't we need to delete  FiscalYears, Ledgers and Funds?

Reply:
It's not possible to delete Fund, Budget, FY, Ledger because of existing Transactions